### PR TITLE
[14.0] [MIG] stock_picking_return_lot

### PR DESCRIPTION
**Pull request by renda-dev**
Migration of `stock_picking_return_lot` from version 12.0 to version 14.0

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
Functional ok

File: stock_picking_return_lot/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module propagates serial numbers or lots form initial picking to backorder.
```
Comment (francesco-ooops) [inline stock_picking_return_lot/readme/DESCRIPTION.rst]: This module propagates serial numbers or lots from initial picking to return picking.

File: stock_picking_return_lot/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Iryna Vyshnevska <i.vyshnevska@mobilunity.com>
```
Comment (francesco-ooops) [inline stock_picking_return_lot/readme/CONTRIBUTORS.rst:1]: Add contributor for migration

**Review by GSLabIt — APPROVED** [review]
code LGTM

**Comment by rousseldenis** [comment]
@renda-dev You should preserve commit history. See : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

**Comment by rousseldenis** [comment]
@renda-dev @GSLabIt @francesco-ooops A little approval there: https://github.com/OCA/stock-logistics-workflow/pull/1635

It fixes current errors in main 14.0 branch.

**Comment by renda-dev** [comment]
> @renda-dev You should preserve commit history. See : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.

Done, let me know if it looks right :)

**Comment by GSLabIt** [comment]
@rousseldenis any chance to double check and merge?

**Comment by GSLabIt** [comment]
@rousseldenis could u please double check and merge? Thx

**Comment by francesco-ooops** [comment]
@pedrobaeza can we ask for merge? thanks!

**Comment by rousseldenis** [comment]
@renda-dev I would say commit history in 12.0 looks like : 

image


Why having removed everything ?

**Comment by francesco-ooops** [comment]
@rousseldenis isn't that what's requested in [the migration guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration), specifically: 

> Squash administrative commits (if any) with the previous commit for reducing commit noise. Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate for details.

?

**Comment by rousseldenis** [comment]
> @rousseldenis isn't that what's requested in [the migration guide](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0#tasks-to-do-in-the-migration), specifically:
> 
> > Squash administrative commits (if any) with the previous commit for reducing commit noise. Check https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate for details.
> 
> ?

This is not mandatory. But ok.

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**